src/redux/data.ts

```diff
@@ -1417,12 +1456,19 @@ export const dataAddNewTransaction = (
           txDetails,
           nativeCurrency
         );
-        const _transactions = [parsedTransaction, ...transactions];
+        const _pendingTransactions = [
+          parsedTransaction,
+          ...pendingTransactions,
```

Comment:
Should  the order of the array elements be the opposite? (considering that parsedTransaction is the most recent data).  It shouldn't make a difference but it's the right thing to do.